components/ingest-service/backend/chef_action.go

```diff
@@ -26,3 +26,27 @@ type InternalChefAction struct {
 	Data             string    `json:"data,omitempty"`
 	Projects         []string  `json:"projects"`
 }
+
+type Indices []struct {
```

Comment:
make it a struct instead of a slice, so that it can be used at other places if needed.